studio/schemas/deskStructure.ts

```diff
@@ -22,13 +23,26 @@ export default (S: StructureBuilder) =>
     .title("Content")
     .items([
       S.listItem()
-        .title("Company Information")
+        .title("Company Details")
```

Comment:
I like "Details" but did we also update the schema and id? if the ID has companyInfo theres bound to be confusion at some point

Reply:
Since this only is the parent section, i guess i dont need to update it in schema